src/UKHO.ERPFacade.Common/Logging/EventIds.cs

```diff
@@ -704,15 +404,39 @@ public enum EventIds
         MessageDequeueCount = 940140,
 
         /// <summary>
-        /// 940141 - Invalid UkhoWeekNumber field in enccontentpublished event from EES.
+        /// 940142 - Exception occurred while decrypting the permit string.
```

Comment:
Please correct the event id order.. Few event sequence numbers are missing.

Reply:
Event Id sequence updated.